Modelica/package.mo

```diff
@@ -2278,7 +2278,7 @@ Graphical illustrations shall not be added in the diagram layer, but can be adde
 </html>"));
     end Icons;
     annotation (DocumentationClass=true,Documentation(info="<html>
-<p>A Modelica main package should be compliant with the UsersGuide stated in this documentation:</p>
+<p>User-facing main packages of the Modelica Standard Library (such as Modelica and Complex) should be compliant with the UsersGuide stated in this documentation:</p>
```

Comment:
Should _Modelica_ and _Complex_ use `<code>` markup here?

Reply:
To me, &lt;em&gt; would be fine here.

Reply:
It's written with `<em>` now.  Resolving.